Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -752,11 +759,12 @@
     - state: icon-chemist
   - type: ItemBorgModule
     items:
-    - HypoBorgMedical # Sunrise-Edit
+    - HypoBorgMedicalAdvanced # Sunrise-Edit
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)